src/main/proto/marathon.proto

```diff
@@ -52,7 +52,6 @@ message ServiceDefinition {
   repeated uint32 ports = 6;
   repeated Constraint constraints = 7;
   required string executor = 8 [default = ""];
-  // optional double taskRateLimit = 9 [default = 1];
```

Comment:
Please don't remove this.  It's here as a reminder to never use index `9` in the future, since this was once a valid index.  This prevents us from misinterpreting the serialized data in case someone upgrades from an ancient (< 1 year ago) version of Marathon.